core/controllers/domain_objects_validator.py

```diff
@@ -186,5 +186,6 @@ def validate_params_dict(params):
     Args:
         params: dict. Data that needs to be validated.
     """
-    for value in params.values():
-        assert isinstance(value, str), ('Expected str, got %s' % value)
+    for key in params.keys():
```

Comment:
Confirm that the type of params to be dict.

Reply:
yes, I found it as `dict` in many files.